criteria.md

```diff
@@ -1,47 +1,83 @@
+| Access Control | The project's [version control system] MUST restrict [collaborator] permissions to the lowest available privileges by default. |
+| Access Control | The project's [version control system] MUST prevent accidental direct [commits] against the [primary branch]. |
+| Access Control | The project's [version control system] MUST prevent accidental deletion of the [primary branch]. |
+| Build & Release | The project's [build and release pipelines] MUST NOT execute [arbitrary code] that is input from outside of the build script. |
+| Build & Release | All [releases] and [released software assets] MUST be assigned a standardized [unique version identifier]. |
 | Build & Release | Any websites related to the project development and release MUST be delivered using SSH or HTTPS. |
-| Documentation | The project MUST have one or more mechanisms for public discussions about proposed changes and usage obstacles. |
-| Documentation | The project documentation MUST provide user guides for all basic functionality. |
-| Documentation | The project documentation MUST include an explanation of the contribution process. |
-| Quality | The version control system MUST be publicly readable and have a static URL. |
-| Quality | The version control system MUST contain a publicly readable record of all changes made, who made the changes, and when the changes were made. |
-| Legal | The version control system MUST require all committers to assert that they are legally authorized to commit the associated contributions on every commit. |
-| Legal | The license for the source code MUST be written in a standardized format approved by the OSI or FSF. |
-| Legal | The license for the source code MUST be maintained in a standard location within the version control system repository. |
-| Legal | The license for the released assets MUST be written in a standardized format approved by the OSI or FSF, if different from the source code license. |
+| Documentation | The project MUST have one or more mechanisms for public discussions about proposed [changes] and usage obstacles. |
+| Documentation | The [project documentation] MUST provide user guides for all basic functionality. |
+| Documentation | The [project documentation] MUST include an explanation of the contribution process. |
+| Quality | The [version control system] MUST be publicly readable and have a static URL. |
+| Quality | The [version control system] MUST contain a publicly readable record of all [changes] made, who made the [changes], and when the [changes] were made. |
+| Legal | The [version control system] MUST require all [code contributors] to assert that they are legally authorized to [commit] the associated contributions on every [commit]. |
+| Legal | The license for the source code MUST be written in a standardized format approved by the [OSI] or [FSF]. |
```

Comment:
I don't understand this one. Is this talking about the format of the `LICENSE` file (or equivalent)? Is it the license identifier in the headers? I assume this is  about using one of the standard license blurbs from OSI, but if so we should specify that this is about the `LICENSE` file.

Reply:
Perhaps "standardized" is a red herring here also.

```suggestion
| Build & Release | All [releases] and [released software assets] MUST be assigned a [unique version identifier]. |
```

Reply:
I'll add a lexicon entry for [license] to clarify this. It should cover both `LICENSE` and `LICENSE/*`.